pkg/services/sqlstore/alert_notification.go

```diff
@@ -246,20 +246,18 @@ func RecordNotificationJournal(ctx context.Context, cmd *m.RecordNotificationJou
 
 func GetLatestNotification(ctx context.Context, cmd *m.GetLatestNotificationQuery) error {
 	return inTransactionCtx(ctx, func(sess *DBSession) error {
-		nj := &m.AlertNotificationJournal{}
+		nj := []m.AlertNotificationJournal{}
```

Comment:
Wonder if we can skip writing success and only insert when "success=true" instead. Then we only need to read the latest successful which makes the code a bit easier to reason about in my mind.

Maybe I'm missing something obvious here?

Reply:
No, your right. 

There might be other reasons to see if notifications fail but for this use case only writing journal events for successful notifications makes sense. 

If we save failed notifications we could list them on the alert notification page or event inform users when % of alert notifications fail.

Reply:
Works a lot better now when starting up :+1: Lets discuss if/how to change this tomorrow. Have to run

Reply:
Did some more testing and enabled reminders. With an empty journal table, having 100 alert rules in alerting state basically behaves the same way as we had initially on startup. I basically got 100 rendering requests at the same time.